### ISSUE TITLE
in the same row, change the value of another cell if actual cell value is changed

### ISSUE THREAD

**Issue by jcabrerazuniga** [AllenFang/react-bootstrap-table#1421]
I am facing an interesting problem:
     For a row R, let's C1 and C2 be cells of R such that the value of C2 is function of the value of C1. In other words, if the value of C1 is changed/updated then C2's value will have to be changed/updated using the new value given to C1. Is there a way to do this using React Boot Strap table's API? 

Thanks

**Comment by AllenFang** [comment]
@jcabrerazuniga sorry for lately reply, could you show me your example? I think it's will more clear.

**Comment by jcabrerazuniga** [comment]
At the next table:
![image](https://user-images.githubusercontent.com/27739794/27743262-9c594ca4-5d78-11e7-8e7a-20a5662ffea9.png)

if I change a cell belonging to the 'Interest rate' column then I need to update the Earnings cell at the same row. For example, if at the first row I change 10% to 25% then the Earnings cell will become 25 and I will need the data to show:

![image](https://user-images.githubusercontent.com/27739794/27743386-152574a0-5d79-11e7-937b-4652acef03bf.png)

So I needed to be able to update another cell after I updated one within the same row. So far I was not be able to do this with the React BootStrap table API. However, I was able to solve this problem making the data table a state on a React class wrapping the React BooStrap Table.
